Mathlib/MeasureTheory/Measure/Haar/Basic.lean

```diff
@@ -747,23 +747,25 @@ see the file `MeasureTheory.Measure.Haar.Unique`.
 
 variable [SecondCountableTopology G]
 
-/-- **Uniqueness of left-invariant measures**: In a second-countable locally compact group, any
+/-- **Uniqueness of left-invariant measures**: In a second-countable topological group, any
```

Comment:
For the record: the current docstring was simply wrong; this is easy to confirm with the API docs.

Reply:
This should be fixed but otherwise looks good!

Reply:
Oh no: I meant "the previous docstring was wrong, the change in this PR fixes it". Sorry for being unclear!

Reply:
I disagree with that comment: the group is assumed to be locally compact, through the existence of a positive compact set in the statement.

Reply:
I defer to @sgouezel !

Reply:
Indeed, I shouldn't try to fix comments after midnight.